app/src/test/java/org/jellyfin/androidtv/util/DeviceUtilsTest.kt

```diff
@@ -0,0 +1,91 @@
+package org.jellyfin.androidtv.util
+
+import android.os.Build
+import io.mockk.*
+import org.junit.After
+import org.junit.Test
+import org.junit.Assert.*
+
+class DeviceUtilsTest {
+	@After
+	fun tearDown(){
+		unmockkAll()
+	}
+
+	@Test
+	fun testBuildModelIsNull(){
```

Comment:
```suggestion
	fun testBuildModelIsNull() {
```
same for other functions